packages/zosfiles/src/api/utils/ZosFilesUtils.ts

```diff
@@ -177,5 +183,50 @@ export class ZosFilesUtils {
         }
         return usspath;
     }
-}
 
+    /**
+     * @param {AbstractSession} session - z/OSMF connection info
+     * @param {string} dataSetName -The name of the data set to recall|migrate
+     * @param {string} returnMessage - Message to rtuern based upon command request
+     * @param {any} hsmCommand - HsmCommand requested
+     *  - If true then the function waits for completion of the request. If false (default) the request is queued.
+     * @param {IRecallOptions | IMigrateOptions} options
+     */
+    public static async dfsmsHsmCommand(
+        session: AbstractSession,
+        dataSetName: string,
+        returnMessage: string,
+        hsmCommand: any,
+        options: Partial<IRecallOptions> | Partial<IMigrateOptions> = {}
+    ): Promise<IZosFilesResponse> {
```

Comment:
Should we create an interface for this function in case we need to expand the number of parameters in the future?

Reply:
We could create an interface, I know that there is still hDelete hsm command that can be added in, in which case creating an interface would make sense

Reply:
Let's try to get an interface when hDelete comes along! (if we still think it's necessary)